networking/hardware_networks/about-sriov.adoc

```diff
@@ -5,12 +5,22 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can use Single Root I/O Virtualization (SR-IOV) network devices with additional networks on your {product-title} cluster for high performance applications.
+Developed by the PCI-SIG (PCI Special Interest Group), the Single Root I/O Virtualization (SR-IOV) specification is a standard for a type of PCI device assignment that can share a single device with multiple pods.
 
-[id="about-using-sr-iov-hardware"]
-== About SR-IOV hardware on {product-title}
+SR-IOV enables you to segment a compliant network device, recognized on the host node as a Physical Function (PF), into multiple Virtual Functions (VFs).
+The VF is used like any other network device.
+The SR-IOV device driver for the device determines how the VF is exposed in the container:
```

Comment:
Hi @jboxman ,  I just realized that my statement about device driver is not accurate, `netdevice` is described as  a `deviceType` in our document, it represents VF that uses its original kernel network driver when first being provisioned. but `netdevice` itself is not a "driver" (for example, there is no driver with "netdevice" name in kernel).

Also given the fact that we are adding InfiniBand device support in SR-IOV Operator, which break the statement here (if Infiniband SR-IOV device is attached to Pod, it may still have a kernel network device attached for IPoIB mode, but the actual InfiniBand device used by application is a character device ).

Given above, maybe we could remove these three sentences for the sake of accuracy,  wdyt?

Reply:
@zshi-redhat, sure. Maybe that kind of information is too detailed for these docs; The Operator is designed to handle those details for the user, anyway.

Reply:
yes,  device plugin and CNI, managed by Operator, make sure those device handlers/specs are attached to container.